service/sharddistributor/config/config.go

```diff
@@ -62,8 +62,10 @@ type (
 
 	Namespace struct {
 		Name string `yaml:"name"`
-		Type string `yaml:"type"` // TODO: this should be and ENUM of fixed/ephemeral
+		Type string `yaml:"type"` // The field is a string since it is shared between global config and also monorepo configs.
```

Comment:
I don't think we should refer to monorepo here, we should probably just say "show be possible to overwrite" 

Which values do we expect this to take ?